cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -135,18 +136,20 @@
 
   vol_array.each_with_index do |volumeid, index|
     dev_path[index] = "/dev/disk/by-ebs-volumeid/#{volumeid}"
-    dev_uuids[index] = get_uuid_for_unmount(shared_dir_array[index])
 
     # Unmount and remove volume from /etc/fstab
-    mount shared_dir_array[index] do
-      device(DelayedEvaluator.new { dev_uuids[index] })
-      fstype(DelayedEvaluator.new { node['cluster']['volume_fs_type'] })
-      device_type :uuid
-      options "_netdev"
-      pass 0
-      action %i(unmount disable)
-      retries 10
+    execute 'unmount ebs' do
+      command "umount -l #{shared_dir_array[index]}"
+      retries 1
       retry_delay 6
+      timeout 30
+      only_if "mountpoint -q #{shared_dir_array[index]}"
```

Comment:
Here and elsewhere.
The `only_if` attribute expects the command to return True or False.
This is not the case for the command `mountpoint`.

Reply:
the only_if and not_if command works also for linux command